prymer/offtarget/bwa.py

```diff
@@ -257,12 +243,6 @@ def __init__(
                               otherwise do not include them.
             threads: the number of threads to use.  If `None`, use all available CPUs.
         """
-        threads = os.cpu_count() if threads is None else threads
-        executable_path = ExecutableRunner.validate_executable_path(executable=executable)
-        self.reverse_complement: bool = reverse_complement
-        self.include_alt_hits: bool = include_alt_hits
-        self.max_hits: int = max_hits
-
         missing_aux_paths = []
         for aux_ext in BWA_AUX_EXTENSIONS:
```

Comment:
Could nice missing index file checking be moved to pybwa?

Reply:
https://github.com/fulcrumgenomics/pybwa/issues/42